Ruddarr/Views/Series/SeriesForm.swift

```diff
@@ -89,6 +92,21 @@ struct SeriesForm: View {
         .tint(.secondary)
     }
 
+    var tagsField: some View {
+        NavigationLink {
+            TagList(selected: $tags)
+                .onChange(of: tags) {
+                    series.tags = Array(tags)
+                }
+        } label: {
```

Comment:
Since Ruddarr is iOS-first, I switched this over to a more native iOS/iPadOS list.

This will break macOS. Feel free to restore that with the `Menu` that you used. Or maybe a `popover()`.